### PR TITLE
Update phone number assignment cmdlets

### PR DESCRIPTION
**Pull request by JensTrier**
Removing Public Preview note from Set/Remove-CsPhoneNumberAssignment
Updating Identity description for Set/Remove-CsPhoneNumberAssignment
Add note about Regional Hosted Meeting for Skype for Business Online tenants
Changed status for Set-CsOnlineApplicationInstance

### CODE REVIEW

File: skype/skype-ps/skype/Set-CsOnlineApplicationInstance.md [hunk 1]
```diff
@@ -15,7 +15,9 @@ ms.reviewer:
 ## SYNOPSIS
 Updates an application instance in Azure Active Directory. 
 
-**Note**: This cmdlet will be deprecated. Use the new [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment) cmdlet instead.
+**Note**: The use of this cmdlet for assigning phone numbers will be deprecated. Use the new [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment) 
+and [Remove-CsPhoneNumberAssignment](/powershell/module/teams/remove-csphonenumberassignment) cmdlets instead. The new cmdlets are currently not supported for customers/tenants
+that are or have been enabled for Regionally Hosted Meetings for Skype for Business Online. These customers should continue to use Set-CsOnlineApplicationInstance.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsOnlineApplicationInstance.md:20]:
```suggestion
and [Remove-CsPhoneNumberAssignment](/powershell/module/teams/remove-csphonenumberassignment) cmdlets instead. The new cmdlets are currently not supported for customers and tenants
who are or have been enabled for Regionally Hosted Meetings for Skype for Business Online. These customers should continue to use Set-CsOnlineApplicationInstance.
```

File: skype/skype-ps/skype/Set-CsOnlineVoiceUser.md [hunk 2]
```diff
@@ -15,7 +15,10 @@ ms.reviewer:
 ## SYNOPSIS
 Use the `Set-CsOnlineVoiceUser` cmdlet to set the PSTN specific parameters (like telephone numbers and emergency response locations.)
 
-**Note**: This cmdlet will be deprecated and no longer supports the ability to add extensions to telephone numbers (for example, "tel:+61281234567;ext=5110"). Use the new [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment) cmdlet instead.
+**Note**: This cmdlet will be deprecated and no longer supports the ability to add extensions to telephone numbers (for example, "tel:+61281234567;ext=5110"). Use the new
+[Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment) and
+[Remove-CsPhoneNumberAssignment](/powershell/module/teams/remove-csphonenumberassignment) cmdlets instead. The new cmdlets are currently not supported for customers/tenants that
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsOnlineVoiceUser.md:20]:
```suggestion
[Remove-CsPhoneNumberAssignment](/powershell/module/teams/remove-csphonenumberassignment) cmdlets instead. The new cmdlets are currently not supported for customers and tenants that
```

File: teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md [hunk 3]
```diff
@@ -15,9 +15,9 @@ schema: 2.0.0
 ## SYNOPSIS
 This cmdlet will assign a phone number to a user or a resource account (online application instance).
 
-> [!NOTE]
-> **Preview** The use of this cmdlet is in Public Preview.
-  
+**Note**: The cmdlet is currently not supported for customers/tenants that are or have been enabled for Regionally Hosted Meetings for Skype for Business Online. These
+customers should continue to use the Set-CsUser, Set-CsOnlineVoiceUser, Set-CsOnlineApplicationInstance or Set-CsOnlineVoiceApplicationInstance cmdlets.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md:19]:
```suggestion
**Note**: The cmdlet is currently not supported for customers and tenants that are or have been enabled for Regionally Hosted Meetings for Skype for Business Online. These
customers should continue to use the Set-CsUser, Set-CsOnlineVoiceUser, Set-CsOnlineApplicationInstance, or Set-CsOnlineVoiceApplicationInstance cmdlets.

```

File: teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md [hunk 4]
```diff
@@ -110,7 +110,8 @@ Accept wildcard characters: False
 ```
 
 ### -Identity
-The Id of the specific user or resource account. Can be specified using the ObjectId, the SIP address or the e-mail address.
+The Identity of the specific user or resource account. Can be specified using the value in the ObjectId, the SipProxyAddress or the UserPrincipalName attribute of the user or
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md:113]:
```suggestion
The Identity of the specific user or resource account. Can be specified using the value in the ObjectId, the SipProxyAddress, or the UserPrincipalName attribute of the user or

```

File: skype/skype-ps/skype/Set-CsOnlineVoiceApplicationInstance.md [hunk 5]
```diff
@@ -15,7 +15,10 @@ ms.author: jenstr
 ## SYNOPSIS
 The `Set-CsOnlineVoiceApplicationInstance` modifies an application instance in Azure Active Directory.
 
-**Note**: This cmdlet will be deprecated. Use the new [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment) cmdlet instead.
+**Note**: This cmdlet will be deprecated. Use the new [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment) and 
+[Remove-CsPhoneNumberAssignment](/powershell/module/teams/remove-csphonenumberassignment) cmdlets instead. The new cmdlets are currently not supported for customers and tenants
+that are or have been enabled for Regionally Hosted Meetings for Skype for Business Online. These customers should continue to use Set-CsOnlineVoiceApplicationInstance.
+
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsOnlineVoiceApplicationInstance.md:22]:
```suggestion
```

File: teams/teams-ps/teams/Remove-CsPhoneNumberAssignment.md [hunk 6]
```diff
@@ -15,8 +15,9 @@ schema: 2.0.0
 ## SYNOPSIS
 This cmdlet will remove/unassign a phone number from a user or a resource account (online application instance).
 
-> [!NOTE]
-> **Preview** The use of this cmdlet is in Public Preview.
+**Note**: This cmdlet is currently not supported for customers and tenants that are or have been enabled for Regionally Hosted Meetings for Skype for Business Online. These
+customers should continue to use the Set-CsUser, Set-CsOnlineVoiceUser, Set-CsOnlineApplicationInstance, or Set-CsOnlineVoiceApplicationInstance cmdlets.
+
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Remove-CsPhoneNumberAssignment.md:20]:
```suggestion
```

File: teams/teams-ps/teams/Remove-CsPhoneNumberAssignment.md [hunk 7]
```diff
@@ -56,7 +57,9 @@ This example removes/unassigns the phone number from user2@contoso.com.
 ## PARAMETERS
 
 ### -Identity
-The Id of the specific user or resource account. Can be specified using the ObjectId, the SIP address or the e-mail address.
+The Identity of the specific user or resource account. Can be specified using the value in the ObjectId, the SipProxyAddress, or the UserPrincipalName attribute of the user or
+
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Remove-CsPhoneNumberAssignment.md:61]:
```suggestion
```

File: teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md [hunk 8]
```diff
@@ -15,9 +15,10 @@ schema: 2.0.0
 ## SYNOPSIS
 This cmdlet will assign a phone number to a user or a resource account (online application instance).
 
-> [!NOTE]
-> **Preview** The use of this cmdlet is in Public Preview.
-  
+**Note**: The cmdlet is currently not supported for customers and tenants that are or have been enabled for Regionally Hosted Meetings for Skype for Business Online. These
+customers should continue to use the Set-CsUser, Set-CsOnlineVoiceUser, Set-CsOnlineApplicationInstance, or Set-CsOnlineVoiceApplicationInstance cmdlets.
+
+
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md:21]:
```suggestion
```

File: teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md [hunk 9]
```diff
@@ -110,7 +111,9 @@ Accept wildcard characters: False
 ```
 
 ### -Identity
-The Id of the specific user or resource account. Can be specified using the ObjectId, the SIP address or the e-mail address.
+The Identity of the specific user or resource account. Can be specified using the value in the ObjectId, the SipProxyAddress, or the UserPrincipalName attribute of the user or
+
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md:115]:
```suggestion
```

File: teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md [hunk 10]
```diff
@@ -184,7 +185,8 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 ### System.Object
 
 ## NOTES
-The cmdlet is available in Teams PS module 2.6.1-preview or later.
+The cmdlet is available in Teams PowerShell module 3.0.0 or later.
+
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md:189]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@JensTrier, thank you for committing my changes. Can you please just remove the extra blank lines that have magically appeared? Many thanks.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@JensTrier, thank you.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Pull request merged**